org.eclipse.xtext.testlanguages/src-gen/org/eclipse/xtext/testlanguages/backtracking/services/BeeLangTestLanguageGrammarAccess.java

```diff
@@ -482,21 +464,13 @@
 		private final Keyword cRightCurlyBracketKeyword_3_2 = (Keyword)cGroup_3.eContents().get(2);
 		
 		//AliasedRequiredCapability:
-		//	(nameSpace=ID | "unit") name=ID ("as" alias=ID)? ('{' ( ('when' ':' condExpr=Expression ';')?  
-		//		& (greedy?="greedy" ';')? 
-		//		& ("requires-min" ':' min=INT ';')? 
-		//		& ("requires-max" ':' max=INT ';')?
-		//		& ("version" ':' versionRange=ID ';')?
-		//	  	)
+		//	(nameSpace=ID | "unit") name=ID ("as" alias=ID)? ('{' (('when' ':' condExpr=Expression ';')? & (greedy?="greedy" ';')
+		//	? & ("requires-min" ':' min=INT ';')? & ("requires-max" ':' max=INT ';')? & ("version" ':' versionRange=ID ';')?)
```

Comment:
Cardinality on new line @ArneDeutsch

Reply:
I have written a test and can not reproduce the behaviour, the question mark remains in the correct line. I assume there is some additional logic taking place (outside the formatter) ensuring that the lines to not get to long. Fear I can not fix this in the formatter.
From this
```
ProvidedCapability: {ProvidedCapability}
	((nameSpace=ID) | "unit") 
	('{'
     	( 	
     	  ('when' ':' condExpr = Expression ';')?
	    & ("name" ':' name = ID ';')          
	    & ("version" ':' version = ID ';')?
	    )
	'}')?
;
```
the formatter produces that
```
ProvidedCapability:
	{ProvidedCapability}
	((nameSpace=ID) | "unit")
	('{'
	(('when' ':' condExpr=Expression ';')? & ("name" ':' name=ID ';') & ("version" ':' version=ID ';')?)
	'}')?;
```

Reply:
What happens if you use two more chars for one of the keywords (to compensate for the '//' indentation string?